### PR TITLE
fix: removed all the attributes of old models 

### LINKED ISSUE #1768: Remove all the attribute names supporting the old api
**Actual Behaviour**

Right now the app has support for old api attributes which needs to be removed now as the new api is live.

**Expected Behaviour**

Remove them.

**Steps to reproduce it**

-

**LogCat for the issue**

-

**Screenshots of the issue**

-

**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

@iamareebjamal please add this.
https://github.com/fossasia/open-event-android/milestone/31

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1768 

Changes: 
removed all attributes of old models and made the models compatible to the new api except the event and speaker model.

Screenshots for the change: 
-

@iamareebjamal @Shailesh351 @shreyasnbhat @rohanagarwal94 
please review.

### CODE REVIEW

File: android/app/src/main/assets/microlocations [hunk 1]
```diff
@@ -1,122 +1,69 @@
-[
+{
```
Comment (iamareebjamal) [inline android/app/src/main/assets/microlocations]: This is incorrect. The API and JSON have a different format. We want to use this format for JSON https://github.com/fossasia/open-event/wiki/%5BNextgen%5D-Open-Event-Universal-Format-Specification

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/assets/microlocations (hunk 1 above)
Comment (iamareebjamal) [reply android/app/src/main/assets/microlocations]: Not the API format

**Comment by arp95** [comment]
@iamareebjamal updated. please see now.

File: android/app/src/main/assets/microlocations [hunk 2]
```diff
@@ -1,122 +1,10 @@
 [
```
Comment (Shailesh351) [inline android/app/src/main/assets/microlocations]: Don't remove sample data. Please change attributes names using script you wrote. Sample should contain more data.

Comment (arp95) [reply android/app/src/main/assets/microlocations]: yes you are right. doing it

**Comment by arp95** [comment]
@iamareebjamal @Shailesh351 
please review now.

File: android/app/src/main/java/org/fossasia/openevent/data/Session.java [hunk 3]
```diff
@@ -28,15 +27,23 @@
...
     private String longAbstract;
     private String comments;
+    @JsonProperty("starts-at")
     private String startTime;
+    @JsonProperty("ends-at")
     private String endTime;
     private String language;
+    @JsonProperty("slides-url")
     private String slides;
+    @JsonProperty("videos-url")
```
Comment (Shailesh351) [inline android/app/src/main/java/org/fossasia/openevent/data/Session.java]: `video-url` it is changed :sweat_smile: 
Please look at other attributes also

**Comment by arp95** [comment]
@iamareebjamal @Shailesh351 @shreyasnbhat @rohanagarwal94 
please review now.

**Comment by Shailesh351** [comment]
@arp95 event sample file also needs to be changed. I think you missed it.

**Comment by arp95** [comment]
@Shailesh351 
actually the event model has a separate pr so i have left the event and speaker model for now as stated above.
please review the other and approve.

**Review by Shailesh351 — APPROVED** [review]
LGTM

**Review by iamareebjamal — DISMISSED** [review]

**Review by rohanagarwal94 — APPROVED** [review]

**Review by Shailesh351 — CHANGES_REQUESTED** [review]
@arp95 In sessions file, not all attributes are changed. Please have a look. Change it accordingly.

**Comment by arp95** [comment]
can you please mention the ones not there. because according to me audio and video attribute changed.

**Comment by Shailesh351** [comment]
@arp95 See https://github.com/arp95/open-event-android/blob/efbca32ab79e22f003efd77897062b3e38eb9914/android/app/src/main/assets/sessions#L55

**Comment by arp95** [comment]
guys please review this and approve.
@Shailesh351 @iamareebjamal

**Review by Shailesh351 — APPROVED** [review]

**Comment by arp95** [comment]
@mariobehling @the-dagger 
please review and merge.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**